src/Microsoft.AspNetCore.Mvc.ViewFeatures/RemoteAttribute.cs

```diff
@@ -33,7 +37,7 @@ public class RemoteAttribute : ValidationAttribute, IClientModelValidator
         /// Intended for subclasses that support URL generation with no route, action, or controller names.
         /// </remarks>
         protected RemoteAttribute()
-            : base(Resources.RemoteAttribute_RemoteValidationFailed)
+            : base(errorMessageAccessor: () => Resources.RemoteAttribute_RemoteValidationFailed)
```

Comment:
Previous approach left `ErrorMessage` non-`null`, disallowing use of `ErrorMessageResourceName` and `ErrorMessageResourceType`.